src/test/scala/day5/CalculatorTest.scala

```diff
@@ -0,0 +1,25 @@
+package day5
+import resources.BaseTestingClass
+
+class CalculatorTest extends BaseTestingClass {
```

Comment:
I would like to see more tests, testing negative values, both numbers 0, one number 0, trying to give a larger number than the method can handle, division by zero etc.